shell-ui/src/auth/permissionUtils.spec.js

```diff
@@ -116,3 +117,21 @@ describe('permission utils - isPathAccessible', () => {
     expect(isAccessible).toBe(false);
   });
 });
+
+describe('permission utils - getUserGroups', () => {
+  it('should return an array of groups when defined in OIDC claims ', () => {
+    //E
+    const oidcGroups = ["oidcGroup"];
+    const groups = getUserGroups({profile: {email: "test@test.com", groups: oidcGroups}}, undefined);
+    //V
+    expect(groups).toEqual(oidcGroups);
+  });
+
+  it('should return a merged array of groups when defined in OIDC claims and mapping ', () => {
+    //E
+    const oidcAndStaticGroups = ["group"];
+    const groups = getUserGroups({profile: {email: "test@test.com", groups: oidcAndStaticGroups}}, {'test@test.com': oidcAndStaticGroups});
+    //V
+    expect(groups).toEqual(oidcAndStaticGroups);
+  });
```

Comment:
Maybe this test case could be improved a bit, by making sure the input OIDC and mapped groups are different (otherwise we're not really testing the "merge" behaviour):
- have one case where OIDC groups are not set at all (to emulate Dex :wink:)
- have another case where there are some duplicates which get discarded by this method